okapi-core/src/test/java/org/folio/okapi/util/ModuleUtilTest.java

```diff
@@ -81,13 +81,77 @@ void testFilterRequireNotify2_1() {
   }
 
   @Test
-  void testFilterRequireNotify2() {
+  void testFilterRequireInstanceStorage() {
     MultiMap params = MultiMap.caseInsensitiveMultiMap();
-    params.set("require", "notify=2");
+    params.set("require", "instance-storage");
+    List<ModuleDescriptor> res = ModuleUtil.filter(params, modulesList, false, false);
+    assertThat(res.size()).isEqualTo(9);
```

Comment:
```suggestion
    assertThat(res).hasSize(9);
```

And after adding toString() to ModuleDescriptor the failure message will be perfect.

Reply:
ok